template_feed/Microsoft.DotNet.Common.ProjectTemplates.1.x/content/ClassLibrary-CoreClr-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,60 @@
+{
+  "author": "Microsoft",
+  "classifications": ["NetCore", "Library"],
+  "name": "Class library",
```

Comment:
This will be confusing to have two class library entries in the list at the CLI. We're working on a feature that should make this set of variations unnecessary. @seancpeters